src/temp/ProjectInfo/SingleProjectInfoItem.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import styles from './ProjectInfo.module.scss';
+
+interface Props {
+  title: ReactElement | string;
+  itemContent: ReactElement;
+}
+
+const SingleProjectInfoItem = ({ title, itemContent }: Props) => {
+  return (
+    <div className={styles.singleInfo}>
+      <div className={styles.halfInfo}>
```

Comment:
Why is this called `halfInfo`?

Reply:
I followed the same structure as before. I'll name it better.